tests/tensorflow/test_sota_checkpoints.py

```diff
@@ -51,6 +51,12 @@
 MO_DIR = OPENVINO_DIR / 'deployment_tools' / 'model_optimizer'
 MO_VENV_DIR = PROJECT_ROOT / 'model_optimizer'
 venv_activate_string = f'source {MO_VENV_DIR}/bin/activate && source {OPENVINO_DIR}/bin/setupvars.sh'
+RC_PACKAGE_NOT_USED = True
```

Comment:
Refactor for this variable to be called `USING_OV2_PACKAGE_FORMAT` and adjust the logic.